site/ko/beta/guide/saved_model.ipynb

```diff
@@ -0,0 +1,1176 @@
+   "source": [
+    "loaded = tf.saved_model.load(\"/tmp/mobilenet/1/\")\n",
+    "print(list(loaded.signatures.keys()))  # [\"serving_default\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "K5srGzowfWff"
+   },
+   "source": [
+    "가져온 시그니처는 항상 딕셔너리를 반환합니다."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "ChFLpegYfQGR"
+   },
+   "outputs": [],
+   "source": [
+    "infer = loaded.signatures[\"serving_default\"]\n",
+    "print(infer.structured_outputs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "cJYyZnptfuru"
+   },
+   "source": [
+    "SavedModel로부터 추론을 실행하면 처음 모델과 같은 결과를 제공합니다."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "9WjGEaS3XfX7"
+   },
+   "outputs": [],
+   "source": [
+    "labeling = infer(tf.constant(x))[pretrained_model.output_names[0]]\n",
+    "\n",
+    "decoded = imagenet_labels[np.argsort(labeling)[0,::-1][:5]+1]\n",
+    "\n",
+    "print(\"저장과 불러오기 이후의 결과:\\n\", decoded)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "SJEkdXjTWbtl"
+   },
+   "source": [
+    "## 모델 배포하기\n",
+    "\n",
+    "SavedModels는 파이썬에서 사용하기에 적합하지만, 일반적으로 프로덕션 환경에서는 추측 전용 서비스를 사용합니다. 이는 TensorFlow Serving을 사용한 SavedModel에서 쉽게 설정할 수 있습니다.\n",
+    "\n",
+    "`tensorflow_model_server`를 노트북이나 로컬 머신에 설치하는 방법을 포함한 배포에 대한 자세한 내용은 [TensorFlow Serving REST 튜토리얼] (https://github.com/tensorflow/serving/blob/master/tensorflow_serving/g3doc/tutorials/Serving_REST_simple.ipynb)을 참조하십시오. 임시방편으로 앞서 내보낸 `mobilenet` 모델을 배포하기 위해 모델 서버를 SavedModel 디렉토리로 설정합니다:\n",
+    "\n",
+    "```bash\n",
+    "nohup tensorflow_model_server \\\n",
+    "  --rest_api_port=8501 \\\n",
+    "  --model_name=mobilenet \\\n",
+    "  --model_base_path=\"/tmp/mobilenet\" >server.log 2>&1\n",
+    "```\n",
+    "\n",
+    "  이제 요청을 보냅니다.\n",
+    "\n",
+    "```python\n",
+    "!pip install requests\n",
+    "import json\n",
+    "import numpy\n",
+    "import requests\n",
+    "data = json.dumps({\"signature_name\": \"serving_default\",\n",
+    "                   \"instances\": x.tolist()})\n",
+    "headers = {\"content-type\": \"application/json\"}\n",
+    "json_response = requests.post('http://localhost:8501/v1/models/mobilenet:predict',\n",
+    "                              data=data, headers=headers)\n",
+    "predictions = numpy.array(json.loads(json_response.text)[\"predictions\"])\n",
+    "```\n",
+    "\n",
+    "`predictions`의 결과는 파이썬에서의 결과와 동일합니다."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "Bi0ILzu1XdWw"
+   },
+   "source": [
+    "### SavedModel 포맷\n",
+    "\n",
+    "SavedModel은 연속된 시그니처와 이를 실행하는 데 필요한 변수값과 어휘를 포함한 상태를 포함하는 디렉토리입니다."
```

Comment:
'serialize' means '직렬화하다' in computer science.
Variable values is included a directory, not state.
I don't know what is vocabularies. But considering the context, it can be written 'variable values and constant values'.
'포함' is repeated. If you want to write well, just avoid repeating words.
I suggest '연속된 시그니처와 이를 실행하는 데 필요한 변수값과 어휘를 포함한 상태를 포함하는 디렉토리입니다'
--> '**변수 값과 상수를 포함하고 직렬화된** 시그니처와 이를 실행하는 데 필요한 상태를 **담은** 디렉토리입니다'